tst/shaka_scheme/system/base/unit-Environment.cpp

```diff
@@ -0,0 +1,200 @@
+//
+// Created by dylan on 9/20/17.
+//
+
+#include <gmock/gmock.h>
+#include "shaka_scheme/system/base/Environment.hpp"
+#include "shaka_scheme/system/base/Data.hpp"
+
+using namespace shaka;
+
+TEST(EnvironmentUnitTest, parent_constructor) {
```

Comment:
Please put Doxygen comments with `@brief` one-line descriptions of the form `@brief Test: blah blah blah` etc. Look at `unit-DataNode.cpp` for an example.